src/reservation/azext_reservation/vendored_sdks/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Perhaps an empty `__init__.py` will work? Since this `vendored_sdks` doesn't need to be a namespace package.

Reply:
To be safe, I copied this file from another extension.
It still works with an empty `__init__.py`